src/Sparrow/LowMemory/MemoryInformation.cs

```diff
@@ -130,7 +130,7 @@ public static bool IsSwappingOnHddInsteadOfSsd()
             return CheckPageFileOnHdd.WindowsIsSwappingOnHddInsteadOfSsd();
         }
 
-        public static bool WillCauseHardPageFault(IntPtr addr, long length) => PlatformDetails.RunningOnPosix ? PosixMemoryQueryMethods.WillCauseHardPageFault(addr, length) : Win32MemoryQueryMethods.WillCauseHardPageFault(addr, length);
+        public static bool WillCauseHardPageFault(IntPtr addr, long length) => PlatformDetails.RunningOnPosix ? !PosixMemoryQueryMethods.WillCauseHardPageFault(addr, length) : !Win32MemoryQueryMethods.WillCauseHardPageFault(addr, length);
```

Comment:
No, this is _REALLY_ confusing. We'll typically look at the impl, not the wrapper